README.md

```diff
@@ -1,3 +1,347 @@
-# Q.U.A.I.L. QUAIL Univeral Analytics of Informatic Linkages
+# Q.U.A.I.L. QUAIL Universal Analytics of Informatic Linkages
 This is a project used to pull down and analyze data from REDCap. It places the project
 metadata and data into sqlite databases arranged by form.
+
+![Python Version](https://img.shields.io/pypi/pyversions/Django.svg)
+![Build Status](https://img.shields.io/codeship/d6c1ddd0-16a3-0132-5f85-2e35c05e22b1.svg)
+
+```
+
+
+
+   ____  _    _         _____ _      
+  / __ \| |  | |  /\   |_   _| |     
+ | |  | | |  | | /  \    | | | |     
+ | |  | | |  | |/ /\ \   | | | |     
+ | |__| | |__| / ____ \ _| |_| |____
+  \___\_\\____/_/    \_\_____|______|
+
+   Q.U.A.I.L. Universal Analytics of Informatic Linkages
+
+
+```
+
+# Installation
+
+ 1. Run ` $ python setup.py install `
```

Comment:
This worked? This is a python module that was meant to be `pip install ./PATH/TO/REPO` to get it to work.

Reply:
yep

Reply:
Ahh. Ok. I understand. Please feel free to fix any of my hackiness with your preferred methods.